.github/workflows/pull-request-testing.yml

```diff
@@ -0,0 +1,23 @@
+name: Pull request testing
+
+on:
+  pull_request
+
+jobs:
+  release:
```

Comment:
fyi, you can have multiple jobs per file, so you can have one workflow for testing on PR and two jobs in it, one for normal testing and another for integration. You do not have to change it, just letting you know in case you didn'